console/my-account/how-to/enforce-mfa-for-organization.mdx

```diff
@@ -0,0 +1,26 @@
+---
+meta:
+  title: How to enforce Multifactor Authentication (MFA) for users in an Organization
+  description: This page explains how to enforce Multifactor Authentication (MFA) for users in an Organization.
+content:
+  h1: How to enforce Multifactor Authentication (MFA) for users in an Organization
+  paragraph: This page explains how to enforce Multifactor Authentication (MFA) for users in an Organization.
+tags: authentication 2FA two-factor two multi factor security google authenticator authenticator
+dates:
+  validation: 2023-06-12
+  posted: 2023-06-12
+categories:
+  - console
+---
+
+Multifactor authentication (MFA) provides extra layers of security. As well as your password (the first factor), other factors (such as a MFA app) are required to gain access to your account. Even in case of leaked password, your account is protected.
+
+You can acitvate the **enforce MFA** feature to ensure all users in your Organization set up MFA. This helps ensure the security of your Organization across all users.  or an Organization you have  have Multifactor Authentication enabled on your Scaleway account for heightened security. Once enabled, an additional security, code generated by your phone, is required to access your account.
+
+Multifactor authentication provides extra layers of security. As well as your password (the first factor), other factors (such as a MFA app) are required to gain access to your account. Even in case of leaked password, your account is protected.
```

Comment:
This paragraph is a repeat of the first one. Although now I'm wondering if this whole file isn't supposed to be here?